lib/mvos/service/firebase_service.dart

```diff
@@ -16,10 +21,29 @@ class FirebaseService implements Disposable {
     FirebaseAuth.instance.signInAnonymously();
   }
 
-  Future<void> loginWithEmailAndPassword() async {
+  Future<void> loginWithEmailAndPassword(String email, String password) async {
     logger.info("Login email and password called");
-    FirebaseAuth.instance.signInWithEmailAndPassword(
-        email: "goranesine@gmail.com", password: "Testpassword01");
+    try {
+      FirebaseAuth.instance
+          .signInWithEmailAndPassword(email: email, password: password);
+    } catch (e) {}
+    return;
```

Comment:
ako je error, trebas `throw Error('objasnjenje errora`)`
Vjerojatno ima par razlicitih errora, tako da mozes napraviti enum sa 
wrong password
mail does not exist
ili tako nesto.

pogledaj kakve errore vraca firebase, i kakvu informaciju moras poslati ekipi u UI da znaju handlati na razlicite nacine

Reply:
Posebni enum dart file?